test/test_utils.py

```diff
@@ -112,6 +112,16 @@ def test_draw_boxes():
     assert_equal(boxes, boxes_cp)
     assert_equal(img, img_cp)
 
+    
+@pytest.mark.parametrize('colors', [
+    None,
+    ['red', 'blue', '#FF00FF', (1, 34, 122)],
+    'red'
```

Comment:
thanks for the test!

could we also check the other kinds of single-color inputs, namely `'#FF00FF'` and `(1, 34, 122)`